web-app/src/pages/resultsPage.js

```diff
@@ -1,15 +1,33 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 import devices from "../styles/Devices";
 import ExperiencesSection from "../components/resultsPage/ExperiencesSection";
 import ResultsSection from "../components/resultsPage/ResultsSection";
+import { getCardIdsFromSessionStorage } from "../util/util";
+import { getResults } from "../services/results.service";
 
 const ResultsPage = () => {
+  const [results, setResults] = useState(undefined);
+  const fetchResults = async () => {
+    let ids = getCardIdsFromSessionStorage();
+    let response = undefined;
+
+    try {
+      response = await getResults(ids);
+      setResults(response ? response : undefined);
+    } catch (exception) {
+      // do nothing
```

Comment:
I think we should log or handle the catch here

Reply:
well logging into browser wont really help us, we should already have a log on the backend since the getResults throws when backend comes back with error code

Reply:
once we have logdna we can do something more fancy